src/main/java/uk/gov/justice/hmpps/nomis/datacompliance/events/dto/OffenderPendingDeletionEvent.java

```diff
@@ -2,7 +2,13 @@
 
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import lombok.*;
+import lombok.AllArgsConstructor;
```

Comment:
different people seem to have different IDE settings around this leading to a bit of ping pong- removing wildcards IS the way to go according to GDS java code guide:

https://gds-way.cloudapps.digital/manuals/programming-languages/java.html#imports